dosagelib/plugins/u.py

```diff
@@ -59,7 +59,7 @@ def getPrevUrl(self, url, data):
         return super(Unsounded, self).getPrevUrl(url, data)
 
     def getIndexStripUrl(self, index):
-        """Get comic strip URL from index."""
+        # Get comic strip URL from index
```

Comment:
Well, since this is just the docstring from the parent class, we could just drop it completly.